### PR TITLE
feat[3.5]: consultations fetching basic data

### PR DESCRIPTION
**Pull request by Pufke**
- [x] Make a simple API endpoint for Consulations, fetch all Consultations

### CODE REVIEW

**Comment by vlaksi** [comment]
Make correct naming 

<details>
  <summary> Your </summary>

image

</details>

<details>
  <summary> Correct ones</summary>

image

</details>


Add Trello card which is connected to this.

File: backend-spring-boot/src/main/java/com/pharmacy/cpis/scheduleservice/controller/ConsultationController.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "api/consulltations")
+public class ConsultationController {
+
```
Comment (vlaksi) [inline backend-spring-boot/src/main/java/com/pharmacy/cpis/scheduleservice/controller/ConsultationController.java:21]: Delete consulltation and put consultation (there is no 2 l in word consultation).

File: backend-spring-boot/src/main/java/com/pharmacy/cpis/util/config/WebSecurityConfig.java [hunk 2]
```diff
@@ -69,6 +69,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers("/auth/**").permitAll()
                 .antMatchers("/h2-console/**").permitAll()
+                .antMatchers("/api/consulltations").permitAll()
```
Comment (vlaksi) [inline backend-spring-boot/src/main/java/com/pharmacy/cpis/util/config/WebSecurityConfig.java:73]: If you are done with this PR, delete this. We do not want public access to this.

Comment (Pufke) [reply backend-spring-boot/src/main/java/com/pharmacy/cpis/util/config/WebSecurityConfig.java:73]: Done

File: backend-spring-boot/src/main/java/com/pharmacy/cpis/scheduleservice/controller/ConsultationController.java (hunk 1 above)
Comment (Pufke) [reply backend-spring-boot/src/main/java/com/pharmacy/cpis/scheduleservice/controller/ConsultationController.java:21]: Done

**Review by vlaksi — APPROVED** [review]
lgtm

**Pull request merged**